packages/utilities/carbon.yml

```diff
@@ -0,0 +1,83 @@
+# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas/carbon-resources.schema.json
+library:
+  id: ibmdotcom-utilities
+  name: IBM.com Utilities
+  description: IBM.com ES6 utility classes
+  externalDocsUrl: https://ibmdotcom-utilities.mybluemix.net
+  noIndex: true
+  demoLinks:
+    - type: storybook
+      name: Storybook
```

Comment:
This library too, use `jsdoc` and `JSDoc`.

Reply:
going for externalDocsUrl instead

Reply:
using externalDocsUrl isntead